actors/abi/sector.go

```diff
@@ -8,6 +8,8 @@ import (
 	cid "github.com/ipfs/go-cid"
 	"github.com/pkg/errors"
 
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
```

Comment:
This will introduce a circular dependency. Please revert.

Reply:
I did not test this code - sorry!